ethereum-consensus/src/capella/spec/mod.rs

```diff
@@ -3312,3 +3313,70 @@ pub fn state_transition<
     state_transition_block_in_slot(state, signed_block, execution_engine, validation, context)
 }
 pub use crate::capella::execution_engine::ExecutionEngine;
+
```

Comment:
if you look at the top of this file you'll see this stuff is automatically generated w/ the `spec-gen` tool

this means you should not be making any edits to it

the `get_expected_withdrawals` function should live next to the `process_withdrawals` function in `crate::capella::block_processing`